packages/hurl/src/http/headers_helper.rs

```diff
@@ -105,4 +118,23 @@ mod tests {
         headers.push(Header::new("content-type", "text/plain"));
         assert_eq!(headers.character_encoding().unwrap().name(), "utf-8");
     }
+
+    #[test]
+    fn aggregate_raw_headers() {
+        let mut headers = HeaderVec::new();
+        headers.push(Header::new("Host", "localhost:8000"));
+
+        let raw_headers = &["User-Agent: hurl/6.0.0", "Invalid-Header"];
+        let aggregated = headers.aggregate_raw_headers(raw_headers);
+
+        assert_eq!(
+            aggregated.get("Host"),
+            Some(&Header::new("Host", "localhost:8000"))
+        );
+        assert_eq!(
+            aggregated.get("User-Agent"),
+            Some(&Header::new("User-Agent", "hurl/6.1.0"))
```

Comment:
Just a small typo here should be `6.0.0` no?

Reply:
When I ran with `--verbose` it did give me `hurl/6.1.0-SNAPSHOT`

Reply:
@jcamiel Oh I see what you mean. That was a oversight haha